lib/jna/clink_jna.cc

```diff
@@ -26,20 +27,30 @@
 extern "C" {
 #endif
 
+tfrt::HostContext *jna_host_context =
```

Comment:
For consistency, could we update all methods in this class (e.g. `SquareAdd`) to use this host context instead of creating a new host context?

And would it be useful to add the `static` keyword here to make sure this variable can only be used by code in this file?

Reply:
OK. I'll make the change to `SquareAdd`.

Besides, I found a bug with the current implementation. `CreateHostContext` relies on some static variables in `static_registration.cc`, which is initialized after these static variables in `clink_jna.cc`. In order to change the initialization order, I'll change it to private methods, like this:
```suggestion
namespace {
tfrt::HostContext *getJnaHostContext() {
  static tfrt::HostContext *jna_host_context =
      clink::CreateHostContext("mstd", tfrt::HostAllocatorType::kMalloc).release();
  return jna_host_context;
}
} // namespace
```

Reply:
Sounds good.